tests/foreman/longrun/test_inc_updates.py

```diff
@@ -0,0 +1,338 @@
+        for act_key in [rhel_6_ak, rhel_6_partial_ak]:
+            act_key.content_override(data={'content_override': {
+                u'content_label': PRD_SETS['rhel6_sat6tools']['label'],
+                u'value': u'1'
+            }})
+
+        # Step 16: Create client machines
+        cls.vm1 = VirtualMachine(distro='rhel67', tag='incupdate')
+        cls.vm2 = VirtualMachine(distro='rhel67', tag='incupdate')
+        cls.vm3 = VirtualMachine(distro='rhel67', tag='incupdate')
+        cls.setup_vm(cls.vm1, rhel_6_ak.name, cls.org.label)
+        cls.setup_vm(cls.vm2, rhel_6_partial_ak.name, cls.org.label)
+        cls.setup_vm(cls.vm3, rhel_6_partial_ak.name, cls.org.label)
+
+        # Find the content hosts and name them
+        cls.contenthost1, cls.contenthost2, cls.contenthost3 = (None, ) * 3
+        systems = System(organization=cls.org).search()
+        for system in systems:
+            if system.name == cls.vm1.hostname:
+                cls.contenthost1 = system
+            elif system.name == cls.vm2.hostname:
+                cls.contenthost2 = system
+            elif system.name == cls.vm3.hostname:
+                cls.contenthost3 = system
+
+    @classmethod
+    def tearDownClass(cls):
+        """Destroys all provisioned vms"""
+        cls.vm1.destroy()
+        cls.vm2.destroy()
+        cls.vm3.destroy()
+        entity_mixins.TASK_TIMEOUT = cls.old_task_timeout
+
+    @staticmethod
+    def setup_vm(client, act_key, org_name):
+        """Creates the vm and registers it to the satellite"""
+        client.create()
+        client.install_katello_cert()
+
+        # Register content host, install katello-agent
+        result = client.register_contenthost(
+            act_key,
+            org_name,
+            releasever='6.7'
+        )
+        assert result.return_code == 0
+        result = client.install_katello_agent()
+        client.run('katello-package-upload')
+        return
+
+    @staticmethod
+    def get_content_view_versions(content_view_id):
+        """Retrieves content view versions for the given content view id"""
+        return ContentView(id=content_view_id).read_json()['versions']
+
+    @staticmethod
+    def get_applicable_errata(repo):
+        """Retrieves applicable errata for the given repo"""
+        return Errata(repository=repo).search(
+            query={'errata_restrict_applicable': True}
+        )
+
+    def test_api_inc_update_noapply(self):
+        """@Test: Check if api incremental update can be done without
+        actually applying it
+
+        @Feature: Incremental Update
+
+        @Assert: Incremental update completed with no errors and Content view
+        has a newer version
+
+        """
+        # Debug statements below.  Used for debugging. Do not remove
+        # self.rhel_6_partial_cv = ContentView(id=38).read()
+        # self.rhel_66_repo = Repository(id=164).read()
+        # self.qe_lce = LifecycleEnvironment(id=46).read()
```

Comment:
How about instructions on what to do instead? Since those IDs would have to be changed anyhow,  you could say something like: To debug blah, make sure to fetch this, that and the other?